src/internal/packager2/layout/import.go

```diff
@@ -175,6 +176,9 @@ func resolveImports(ctx context.Context, pkg v1alpha1.ZarfPackage, packagePath,
 
 func validateComponentCompose(c v1alpha1.ZarfComponent) error {
 	errs := []error{}
+	if strings.Contains(c.Import.Path, v1alpha1.ZarfPackageTemplatePrefix) || strings.Contains(c.Import.URL, v1alpha1.ZarfPackageTemplatePrefix) {
```

Comment:
As a part of the packager2 SDK I appreciate this. Can I confirm though that you won't actually hit this code from the CLI as linting will always occur before? For `package create` I believe this is true but want to make sure I'm not overlooking a potential workflow.

Reply:
My original description wasn't very clear, I reworded it. Users can hit this error now as linting happens after compose. This also means that users were not hitting this error before, so the error message wouldn't directly suggest that package templates were not rendered in the workflow

Reply:
Thanks for the clarity. Part of my issue was testing locally with `path: ###ZARF_PKG_TMPL_PATH###` instead of having it properly wrapped `path: "###ZARF_PKG_TMPL_PATH###"`